src/types/index.ts

```diff
@@ -32,3 +32,34 @@ export type SearchArtist = {
   totalLikes: number;
   followed: boolean;
 };
+
+export interface User {
```

Comment:
키워드를 type으로 통일하는 게 좋을 것 같습니다
```suggestion
export type User = {
```